kubernetes-model/kubernetes-model-core/src/main/java/io/fabric8/kubernetes/api/model/KubernetesList.java

```diff
@@ -43,7 +44,7 @@
 })
 @JsonDeserialize(using = JsonDeserializer.None.class)
 @Buildable(editableEnabled = false, validationEnabled = false, generateBuilderPackage=true, builderPackage = "io.fabric8.kubernetes.api.builder", inline = @Inline(type = Doneable.class, prefix = "Doneable", value = "done"))
-public class KubernetesList extends BaseKubernetesList implements KubernetesResource {
+public class KubernetesList extends BaseKubernetesList implements Serializable {
```

Comment:
Why was this changed?

Reply:
looks like a mistake. I would revert it to it's previous state